packages/web/src/components/tracks-table/TracksTable.tsx

```diff
@@ -305,7 +321,7 @@ export const TracksTable = ({
         track.is_delete || track._marked_deleted || !!track.user?.is_deactivated
       const isOwner = track.owner_id === userId
       if (isLocked || deleted || isOwner) {
-        return <div className={styles.placeholderButton} />
+        return null
```

Comment:
Had to modify the container behavior here. Previously these buttons were being rendered as empty divs whenever they're not there. It was causing stuff to overflow now that we have one more potential thing to render. 
Modified the container to just use gap/margin/flex positioning

Reply:
nice seems like an improvement